.gitlab-ci.yml

```diff
@@ -71,7 +71,7 @@ ci-authorized:
 
 include:
   - .gitlab-ci/lint.yml
-  - .gitlab-ci/shellcheck.yml
+  # - .gitlab-ci/shellcheck.yml
```

Comment:
Why did you comment shellcheck ? If this was because of issues, the CI is now fine in master you may rebase

Reply:
Yeah, there were some issues it was reporting which seemed to be caused by other branches / tags, so I commented to work around it temporarily. I've removed from history and force pushed, so let's see how we get on now